Automattic/wp-calypso
# Add educational content to the cancellation flow

#### Proposed Changes

As a part of the cancellation flow project( pebzTe-kL-p2 ), this PR adds educational content to the cancellation flow. The new content will be shown when a user picks specific answers. For more dedtails, check out pdZzpt-e9-p2.

_Note that this PR is based on #69567. You should merge the preceding PR first._

#### Testing Instructions



* Attempt to cancel your paid plan.
* When you choose `Technical issues > SEO issues`, you should be able to see the educational content.
  <img width="1009" alt="Cancel_Purchase_‹_Taggon_on_wordpress_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/201336269-3f96f3ed-35a5-4f1f-947a-72c22e43d48a.png">
* The open question form will show up if you click on the `Cancel my current plan` button.

Other educational resources will like as follows:
<img width="687" alt="Cancel_Purchase_‹_Taggon_on_wordpress_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/201336463-5714d8bc-62c7-4d4f-a503-eeb0875afbe2.png">
<img width="649" alt="Cancel_Purchase_‹_Taggon_on_wordpress_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/201336733-63bc87a4-5fba-4d24-8bcc-74e1192b53fb.png">
<img width="446" alt="Cancelation_flow_–_Figma" src="https://user-images.githubusercontent.com/212034/202840147-60b9eabb-99a8-4f74-a035-eea75697acee.png">


#### Pre-merge Checklist





Related to #
